GalaxyZooTouchTable/GalaxyZooTouchTable/Views/ExamplesPanel.xaml

```diff
@@ -133,128 +316,238 @@
+                <Border Grid.RowSpan="2" Grid.ColumnSpan="3" Background="{StaticResource MedGrayColor}" CornerRadius="0,5,0,0">
+                    <i:Interaction.Triggers>
+                        <i:EventTrigger EventName="TouchDown">
+                            <i:InvokeCommandAction Command="{Binding TogglePanel}"/>
+                        </i:EventTrigger>
+                    </i:Interaction.Triggers>
+                </Border>
+
+                <Grid Grid.Row="1" Grid.Column="1" RenderTransformOrigin="0,1" Style="{StaticResource HeaderGrid}">
+                    <i:Interaction.Triggers>
+                        <i:EventTrigger EventName="TouchDown">
+                            <i:InvokeCommandAction Command="{Binding TogglePanel}"/>
+                        </i:EventTrigger>
+                    </i:Interaction.Triggers>
+
                     <Grid.RenderTransform>
                         <TransformGroup>
                             <RotateTransform x:Name="RotateHeader" Angle="90"/>
                             <TranslateTransform x:Name="TranslateHeader" X="{Binding Path=ActualWidth, ElementName=Panel, Converter={StaticResource HidePanelConverter}, ConverterParameter='0.66'}"/>
                         </TransformGroup>
                     </Grid.RenderTransform>
-                    
+
                     <Grid.RowDefinitions>
                         <RowDefinition Height="1*"/>
                         <RowDefinition Height="1*"/>
                     </Grid.RowDefinitions>
 
-                    <Viewbox HorizontalAlignment="Left">
-                        <TextBlock Text="Examples" Foreground="White" FontFamily="/GalaxyZooTouchTable;component/Fonts/#Karla" FontWeight="Bold"/>
-                    </Viewbox>
+                    <TextBlock Text="Examples" Foreground="White" FontFamily="/GalaxyZooTouchTable;component/Fonts/#Karla" FontWeight="Bold"/>
 
                     <Viewbox Grid.Row="1" Margin="0,0,10,0">
                         <TextBlock Text="Tap each item to learn more." Foreground="White" FontFamily="/GalaxyZooTouchTable;component/Fonts/#Karla" FontStyle="Italic"/>
                     </Viewbox>
                 </Grid>
 
-                <Grid Grid.Row="1" Grid.Column="1" Panel.ZIndex="1" Style="{StaticResource SelectedItemGrid}">
+                <Grid x:Name="ExampleGrid" Grid.Column="1" Grid.Row="2">
                     <Grid.RowDefinitions>
-                        <RowDefinition Height="1*"/>
+                        <RowDefinition Style="{StaticResource ActiveExample}" Height="1*"/>
                         <RowDefinition Height="2*"/>
                     </Grid.RowDefinitions>
 
-                    <Border Grid.Row="0" Margin="2" x:Name="SelectedElement"/>
+                    <Border Style="{StaticResource SmoothExampleBorder}" BorderThickness="0 1 0 0" BorderBrush="{StaticResource DarkGrayColor}">
```

Comment:
These three `Border`s (`SmoothExampleBorder`, `FeaturesExampleBorder`, and `StarExampleBorder`) are the one place where I have some repetitive code I could _maybe_ refactor into a template. Note, the borders are the same with small differences in the bindings and Y render transform locations.

I'll continue to play with this a bit, but the rest is good for a complete review.